src/frontend/components/SøknadsSteg/OmBarnaDine/OmBarnaDine.tsx

```diff
@@ -52,7 +53,9 @@ const OmBarnaDine: React.FC = () => {
                             id={omBarnaDineSpråkTekstId[OmBarnaDineSpørsmålId.hvemErFosterbarn]}
                         />
                     }
-                    felt={skjema.felter.hvemErFosterbarn}
+                    skjemafelt={skjema.felter.hvemErFosterbarn}
+                    søknadsdatafelt={barnDataKeySpørsmål.erFosterbarn}
+                    nullstillValgteBarn={skjema.felter.erNoenAvBarnaFosterbarn.verdi === ESvar.NEI}
```

Comment:
Hm, tok meg et sekund å skjønne at man nullistilte for akkurat denne gruppa og ikke hvilke barn som er med i søknaden. Vet ikke om det sier noe om prop-navnet eller hjernen min :)